parler/models.py

```diff
@@ -321,8 +321,11 @@ def _set_translated_fields(self, language_code=None, **fields):
             )
             for field, value in model_fields.items():
                 try:
-                    setattr(translation, field, value)
-                except TypeError:
+                    # Fixed FileField clearing.
+                    # See: https://github.com/django-parler/django-parler/issues/326
+                    model_field = parler_meta.model._meta.get_field(field)
+                    model_field.save_form_data(translation, value)
```

Comment:
Is this a Django or Parler API?

Reply:
This is Django's API, for the related "field".  In this case, we need it because the FileField does special case logic on the Django side that needs to be accounted for (handles the special case "False").

![image](https://github.com/user-attachments/assets/05260af3-3f0b-4237-b20a-3ce66eeb4dbb)